lib/membrane_rtc_engine/engine.ex

```diff
@@ -413,6 +414,17 @@ defmodule Membrane.RTC.Engine do
     :ok
   end
 
+  @doc """
+  Sends message to RTC Engine endpoint.
+  If endpoint doesn't exist message is ignored
+  """
```

Comment:
```suggestion
  @doc """
  Sends message to an endpoint.
  
  If endpoint doesn't exist message is ignored.
  """
```

Could we add test for this? In particular, test where we send messsage to non existing endpoint.